src/components/Scheme/ContributionRewardExtRewarders/Competition/utils.ts

```diff
@@ -105,7 +111,9 @@ export interface ICreateSubmissionOptions {
   tags: Array<string>;
 }
 
-export const createCompetitionSubmission = (proposalId: string, options: ICreateSubmissionOptions ): ThunkAction<any, IRootState, null> => {
+const standardPolling = (poll: boolean, fetchAllData = false) => { return { polling: poll, pollInterval: GRAPH_POLL_INTERVAL, fetchAllData }; };
```

Comment:
does this not defined somewhere else ?m lib/util ?

Reply:
this is a different version of the function that is only needed in this one module.